x/wasm/types/keys_test.go

```diff
@@ -29,12 +29,12 @@ func TestGetContractByCodeIDSecondaryIndexPrefix(t *testing.T) {
 }
 
 func TestGetContractByCreatedSecondaryIndexKey(t *testing.T) {
-	c := &ContractInfo{
+	e := ContractCodeHistoryEntry{
 		CodeID:  1,
-		Created: &AbsoluteTxPosition{2 + 1<<(8*7), 3 + 1<<(8*7)},
```

Comment:
I would keep a different `Created` here just to prove it takes one and ignores the other.

Reply:
I have switched from `ContractInfo` to `ContractCodeHistoryEntry` element. Any instantiation/migration results in a new `ContractCodeHistoryEntry` so that it covers both.